### PR TITLE
Dex_Finance_Vault

### PR DESCRIPTION
**Pull request by 0xpeluche**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/agentfi exports TVL: 
        
 
 
    blast                     238.73 k
    
    total                    238.73 k

**Comment by llamatester** [comment]
The adapter at projects/thetanuts exports TVL: 
        
 
 
    ethereum                  15.10 M
    arbitrum                  10.06 M
    fantom                    750.37 k
    bsc                       213.15 k
    polygon                   113.02 k
    boba                      43.68 k
    avax                      36.62 k
    polygon_zkevm             15.37 k
    aurora                    7.19 k
    cronos                    3.10 k
    inevm                     600
    
    total                    26.34 M

**Comment by llamatester** [comment]
The adapter at projects/dexfinance-vault exports TVL: 
        
 
 
    base                      1.75 M
    pulse                     641.29 k
    arbitrum                  199.61 k
    bsc                       186.93 k
    avax                      12.01 k
    optimism                  3.09 k
    fantom                    1.14 k
    
    total                    2.79 M

File: projects/dexfinance-vault/index.js [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+  }, {});
+};
+
+const lpv2Balances = async (api, farms) => {
+  farms.forEach(({ stakingToken, liquidity }) => {
+    api.add(stakingToken, liquidity);
+  });
+};
+
+const lpv3Balances = async (api, farms) => {
```
Comment (g1nt0ki) [inline projects/dexfinance-vault/index.js:92]: why not use the code with positionIds? rest of the PR is good

**Comment by llamatester** [comment]
The adapter at projects/agentfi exports TVL: 
        
 
 
    blast                     239.49 k
    
    total                    239.49 k

**Comment by llamatester** [comment]
The adapter at projects/thetanuts exports TVL: 
        
 
 
    ethereum                  15.01 M
    arbitrum                  10.02 M
    fantom                    740.81 k
    bsc                       211.21 k
    polygon                   112.47 k
    boba                      43.63 k
    avax                      36.29 k
    polygon_zkevm             15.28 k
    aurora                    7.16 k
    cronos                    3.05 k
    inevm                     236
    
    total                    26.20 M

File: projects/dexfinance-vault/index.js (hunk 1 above)
Comment (0xpeluche) [reply projects/dexfinance-vault/index.js]: I'm using it right now, it works well :)

**Comment by llamatester** [comment]
The adapter at projects/dexfinance-vault exports TVL: 
        
 
 
    base                      1.72 M
    pulse                     637.46 k
    arbitrum                  196.64 k
    bsc                       183.77 k
    avax                      11.82 k
    optimism                  3.08 k
    fantom                    1.13 k
    
    total                    2.76 M

**Comment by llamatester** [comment]
The adapter at projects/agentfi exports TVL: 
        
 
 
    blast                     239.40 k
    
    total                    239.40 k

**Comment by llamatester** [comment]
The adapter at projects/thetanuts exports TVL: 
        
 
 
    ethereum                  15.00 M
    arbitrum                  9.95 M
    fantom                    740.03 k
    bsc                       211.05 k
    polygon                   112.23 k
    boba                      43.56 k
    avax                      36.27 k
    polygon_zkevm             15.25 k
    aurora                    7.16 k
    cronos                    3.05 k
    inevm                     236
    
    total                    26.12 M

**Pull request merged**